ci-operator/step-registry/aws/deprovision/stacks/aws-deprovision-stacks-commands.sh

```
@@ -14,15 +14,35 @@ fi
 function delete_stacks()
 {
     local stack_list=$1
+    local max_retries=3
     for stack_name in `tac ${stack_list}`; do 
         echo "Deleting stack ${stack_name} ..."
-        aws --region $REGION cloudformation delete-stack --stack-name "${stack_name}" &
-        wait "$!"
-        echo "Deleted stack ${stack_name}"
+        local attempt=1
+        while [ $attempt -le $max_retries ]; do
+            aws --region $REGION cloudformation delete-stack --stack-name "${stack_name}" &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)